lib/gettext-setup/gettext_setup.rb

```diff
@@ -38,6 +41,24 @@ def self.initialize(locales_path, options = {})
     Locale.set_default(default_locale)
   end
 
+  def self.build_translation_repository(project_name, options)
+    FastGettext::TranslationRepository.build(project_name,
+                                             :path => locales_path,
+                                             :type => options[:file_format] || :po,
+                                             :ignore_fuzzy => false)
+  end
+
+  def self.add_repository_to_chain(repository)
+    @@translation_repositories.push(repository)
+  end
+
+  def self.chain_contains?(repository)
+    @@translation_repositories.each do |in_chain|
+      return true if repository.instance_variable_get(:@name) == in_chain.instance_variable_get(:@name)
```

Comment:
I wasnt sure what else to do. Ask fastgettext to expose the name as a method? why not?